.github/workflows/deploy.yml

```diff
@@ -51,7 +51,7 @@ jobs:
 
     - uses: actions/upload-artifact@v4
       with:
-        name: cibw-wheels-${{ matrix.os }}-${{ strategy.job-index }}
```

Comment:
I don't understand what's happening here fully, I would make sure to test this.

Reply:
The `-${{ strategy.job-index }}` was an necessary number for the artefact name, e.g. "cibw-wheels-macos-latest-2.zip". So now it becomes just "cibw-wheels-macos-latest.zip". Artefacts can be viewed/downloaded in the GHA Summary, which I sometimes download to inspect or upload to PyPI (hopefully less-so for future releases).

Reply:
Doesn't this give a problem with macOS artifact names? Since cibuildwheel runs both on x86 and Arm runners?

Reply:
I just checked to see, and the files look good. Each artefact contains several wheel files created from the runner, so there's no conflict.